src/main/java/teammates/storage/api/FeedbackResponsesDb.java

```diff
@@ -484,29 +514,41 @@ private FeedbackResponse getFeedbackResponseEntity(
                 .first().now();
     }
 
-    private List<FeedbackResponse> getFeedbackResponseEntitiesForQuestionInSection(
-                String feedbackQuestionId, String section) {
+    private List<FeedbackResponse> getFeedbackResponseEntitiesForQuestionInSections(
+                String feedbackQuestionId, List<String> sections, SectionDisplayMode sectionDisplayMode) {
         List<FeedbackResponse> feedbackResponses = new ArrayList<>();
 
-        feedbackResponses.addAll(load()
-                .filter("feedbackQuestionId =", feedbackQuestionId)
-                .filter("giverSection =", section)
-                .filter("receiverSection =", section)
-                .list());
+        SectionDisplayMode displayMode = sectionDisplayMode == null
+                ? SectionDisplayMode.GIVER_OR_RECIPIENT_IN_SECTION
```

Comment:
Might be better to extract this to `SectionDisplayMode`, e.g. `SectionDisplayMode.getDefaultValue()`.